comm/src/main/scala/coop/rchain/comm/rp/Connect.scala

```diff
@@ -29,9 +29,9 @@ object Connect {
   object ConnectionsCell {
     def apply[F[_]](implicit ev: ConnectionsCell[F]): ConnectionsCell[F] = ev
 
-    def random[F[_]: Monad: ConnectionsCell: RPConfAsk]: F[Connections] =
+    def random[F[_]: Monad: ConnectionsCell: RPConfAsk](itemsNum: Int = 0): F[Connections] =
       for {
-        max   <- RPConfAsk[F].reader(_.maxNumOfConnections)
+        max   <- if (itemsNum == 0) RPConfAsk[F].reader(_.maxNumOfConnections) else itemsNum.pure[F]
```

Comment:
Can we remove `RPConfAsk` from here and use it in `CommUtil` as default. Maybe `Option[Int]` for default would be more clear.